plugins/modules/azure_rm_virtualnetworkpeering.py

```diff
@@ -350,7 +408,8 @@ def create_or_update_vnet_peering(self):
             allow_virtual_network_access=self.allow_virtual_network_access,
             allow_gateway_transit=self.allow_gateway_transit,
             allow_forwarded_traffic=self.allow_forwarded_traffic,
-            use_remote_gateways=self.use_remote_gateways)
+            use_remote_gateways=self.use_remote_gateways
+            )
```

Comment:
closing bracket does not match indentation of opening bracket's line
```suggestion
        )
```

Reply:
Fixed indentation.